lib/graphql/resolvers.ts

```diff
@@ -7,25 +7,27 @@ import { permits, createPermit } from '@lib/permits/resolvers';
 import { IFieldResolver } from 'graphql-tools'; // GraphQL field resolver
 import { Context } from '@lib/context'; // Context type
 import { dateScalar } from '@lib/scalars'; // Custom date scalar implementation
+import { authenticate } from '@lib/auth-guards';
+import { Role } from '@lib/graphql/types';
 
 // Resolver type
 export type Resolver<P = undefined> = IFieldResolver<P, Context>;
 
 const resolvers = {
```

Comment:
Would be great if you could add a small comment above this line briefly explaining the usage of `authenticate` here to protect the routes and restrict them to RCD employee roles.